src/essreflectometry/reflectometry/types.py

```diff
@@ -0,0 +1,67 @@
+from typing import NewType, TypeVar
+
+import sciline
+import scipp as sc
+
+Reference = NewType('Reference', str)
+Sample = NewType('Sample', str)
+Run = TypeVar('Run', Reference, Sample)
+
+
+class Raw(sciline.Scope[Run, sc.DataArray], sc.DataArray):
```

Comment:
```suggestion
class RawData(sciline.Scope[Run, sc.DataArray], sc.DataArray):
```
For more symmetry with the names in the next steps.